### PR TITLE
Additional tests to tasks 086a, 086b, and 330 have been added.

### PR DESCRIPTION
**Pull request by MykhailoGrynenko**

### CODE REVIEW

File: Tasks/test_task_086a.py [hunk 1]
```diff
@@ -8,3 +11,28 @@ def test_task_086a():
     assert task_086a.run_body(1575) == 4
     assert task_086a.run_body(2021) == 4
     assert task_086a.run_body(12345) == 5
+
+
+class TestTask086a(TestCase):
+    def test_task_559(self) -> None:
```
Comment (CheshirLvova) [inline Tasks/test_task_086a.py]: change name of the function according to your tests

File: Tasks/test_task_086b.py [hunk 2]
```diff
@@ -8,3 +12,28 @@ def test_task_086a():
     assert task_086b.run_body(1575) == 18
     assert task_086b.run_body(2021) == 5
     assert task_086b.run_body(12345) == 15
+
+
+class TestTask086b(TestCase):
+    def test_task_559(self) -> None:
```
Comment (CheshirLvova) [inline Tasks/test_task_086b.py]: change name of the function according to your tests

File: Tasks/test_task_330b.py [hunk 3]
```diff
@@ -11,3 +15,31 @@ def test_task_330():
     assert task_330.run_body(497) == [6, 28, 496]
     assert task_330.run_body(10000) == [6, 28, 496, 8128]
     assert task_330.run_body(100000) == [6, 28, 496, 8128]
+
+
+class TestTask330(TestCase):
+    def test_task_559(self) -> None:
```
Comment (CheshirLvova) [inline Tasks/test_task_330b.py]: change name of the function according to your tests

File: Tasks/test_task_086b.py [hunk 4]
```diff
@@ -8,3 +12,28 @@ def test_task_086a():
...
+        test1 = task_086b.run_body(2)
+        self.assertEqual(test1, 2)
+
+        with pytest.raises(TaskError):
```
Comment (CheshirLvova) [inline Tasks/test_task_086b.py:22]: 0 won't call TaskError

File: Tasks/test_task_086a.py [hunk 5]
```diff
@@ -8,3 +11,28 @@ def test_task_086a():
...
+        test1 = task_086a.run_body(2)
+        self.assertEqual(test1, 1)
+
+        with pytest.raises(TaskError):
```
Comment (CheshirLvova) [inline Tasks/test_task_086a.py:21]: 0 won't call task error

File: Tasks/test_task_086a.py [hunk 6]
```diff
@@ -8,3 +11,28 @@ def test_task_086a():
...
+        test4 = task_086a.run_body(14)
+        self.assertEqual(test4, 2)
+
+        test5 = task_086a.run_body(3444)
+        self.assertEqual(test5, 4)
+
+        with pytest.raises(TypeError):
+            task_086a.run_body("six")
+
+        with pytest.raises(TaskError):
```
Comment (CheshirLvova) [inline Tasks/test_task_086a.py:37]: here is also an exception would be call, so there would be assertEqual
(and after that you won't need module

File: Tasks/test_task_086b.py [hunk 7]
```diff
@@ -8,3 +12,28 @@ def test_task_086a():
...
+        test4 = task_086b.run_body(14)
+        self.assertEqual(test4, 5)
+
+        test5 = task_086b.run_body(3444)
+        self.assertEqual(test5, 15)
+
+        with pytest.raises(TypeError):
+            task_086b.run_body("six")
+
+        with pytest.raises(TaskError):
```
Comment (CheshirLvova) [inline Tasks/test_task_086b.py:38]: exception

File: Tasks/test_task_330b.py [hunk 8]
```diff
@@ -11,3 +15,31 @@ def test_task_330():
...
+        test1 = task_330.run_body(2)
+        self.assertEqual(test1, [])
+
+        with pytest.raises(TaskError):
```
Comment (CheshirLvova) [inline Tasks/test_task_330b.py]: exception

File: Tasks/test_task_330b.py [hunk 9]
```diff
@@ -11,3 +15,31 @@ def test_task_330():
...
+        test5 = task_330.run_body(29)
+        self.assertEqual(test5, [6, 28])
+
+        test6 = task_330.run_body(99999)
+        self.assertEqual(test6, [6, 28, 496, 8128])
+
+        with pytest.raises(TypeError):
+            task_330.run_body("six")
+
+        with pytest.raises(TaskError):
```
Comment (CheshirLvova) [inline Tasks/test_task_330b.py]: exception

**Review by CheshirLvova — CHANGES_REQUESTED** [review]

File: Tasks/test_task_086a.py (hunk 1 above)
Comment (MykhailoGrynenko) [reply Tasks/test_task_086a.py]: Thank you. You are quite perceptive.

File: Tasks/test_task_086b.py (hunk 2 above)
Comment (MykhailoGrynenko) [reply Tasks/test_task_086b.py]: Thank you.

File: Tasks/test_task_086b.py (hunk 4 above)
Comment (MykhailoGrynenko) [reply Tasks/test_task_086b.py:22]: I'm sorry, I'm afraid, I don't understand you.
I added another test:
        with pytest.raises(TaskError):
            task_086b.run_body(0)
Thank you for your feedback!

File: Tasks/test_task_086a.py (hunk 5 above)
Comment (v-lavrushko) [reply Tasks/test_task_086a.py:21]: is this a comment to test, or task itself?)

File: Tasks/test_task_086a.py [hunk 10]
```diff
@@ -8,3 +11,31 @@ def test_task_086a():
     assert task_086a.run_body(1575) == 4
     assert task_086a.run_body(2021) == 4
     assert task_086a.run_body(12345) == 5
+
+
+class TestTask086a(TestCase):
+    def test_task_330(self) -> None:
```
Comment (CheshirLvova) [inline Tasks/test_task_086a.py]: not 330, but 086a

File: Tasks/test_task_086a.py (hunk 5 above)
Comment (CheshirLvova) [reply Tasks/test_task_086a.py:21]: hehe, to test)

File: Tasks/test_task_086b.py [hunk 11]
```diff
@@ -8,3 +12,31 @@ def test_task_086a():
     assert task_086b.run_body(1575) == 18
     assert task_086b.run_body(2021) == 5
     assert task_086b.run_body(12345) == 15
+
+
+class TestTask086b(TestCase):
+    def test_task_330(self) -> None:
```
Comment (CheshirLvova) [inline Tasks/test_task_086b.py]: not 330, but 086b

**Review by CheshirLvova — CHANGES_REQUESTED** [review]
change  titles of some funcs

File: Tasks/test_task_086a.py (hunk 10 above)
Comment (MykhailoGrynenko) [reply Tasks/test_task_086a.py]: Thanks.

File: Tasks/test_task_086b.py (hunk 11 above)
Comment (MykhailoGrynenko) [reply Tasks/test_task_086b.py]: Thanks.

**Review by OlehHnyp — APPROVED** [review]

**Review by CheshirLvova — APPROVED** [review]
everything ok

**Review by ProkhvatylovArtem — APPROVED** [review]

**Review by ruslanliska — APPROVED** [review]

**Pull request merged**